docker-compose-tests.yaml

```diff
@@ -5,12 +5,35 @@ networks:
     name: itu-minitwit-network
 
 services:
+  minitwit-database:
+    image: postgres:16-alpine
+    container_name: psql
+    networks:
+      - main
+    environment:
+      POSTGRES_USER: minitwit
+      POSTGRES_DB: minitwit_test
+      POSTGRES_PASSWORD: '1234'
+    ports:
+      - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
```

Comment:
Can you explain what this does?
Mainly because I see a different syntax on this source:
https://medium.com/@saklani1408/configuring-healthcheck-in-docker-compose-3fa6439ee280

Reply:
Just took it from this link:
https://github.com/peter-evans/docker-compose-healthcheck

Idea is that it runs this command to check if the service is ready

Reply:
Looks good